ftw/contacts/resources/contactfolder_listing.js

```diff
@@ -0,0 +1,81 @@
+$(function() {
+    ContactFolderListing.init();
```

Comment:
Javascript goes from top to bottom and created the object structure first before the first function call. The definition for ``` ContactFolderListing ``` is invoked immediately. jQuery's ready event is fired later. So in my opinion there should be no problem with the loading order.